Examples/Modules/boosted_diags/inputs_3d_slice_reformatted

```diff
@@ -0,0 +1,123 @@
+warpx.zmax_plasma_to_compute_max_step = 0.0031
```

Comment:
Do we need both this and `inputs_3d_slice`?

Reply:
We could remove the old one actually. The old one tests the old implementation and new tests the new implementation. Will do whatever you suggest :)

Reply:
I don't think it makes a big difference but IMO there's slightly less chance of confusion if there's only one inputs file.